### PR TITLE
BERT Tokenizer

### PR DESCRIPTION
**Pull request by gmarceaucaron**
A utils class for dataset analysis.
At this moment, it only creates a list of unique tokens from all data sources specified in a file.
This list is then saved in a text file.
I also added, by accident, a dependence to SentencePiece to the requirements.txt.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 flake8
 pytest
 tensorflow
+sentencepiece
```
Comment (jerpint) [inline requirements.txt]: Remove this

**Comment by jerpint** [comment]
TODO:

* Update readme with instructions on how to use this PR
* Remove bash files (if they are no longer necessary)
* Set the default download directory to `data/`

**Comment by afansi** [comment]
ok. seems goods to me!

**Comment by jerpint** [comment]
@gmarceaucaron can you add instructions on how to use this code?

**Comment by gmarceaucaron** [comment]
I renamed the branch for reducing the scope of it to only the BERT tokenizer.
I will remove the useless code about SentencePair tokenizer.
Is there any unit test we can do concerning the BERT tokenizer?

**Comment by gmarceaucaron** [comment]
It is relevant to use the functions check_answer_tokenization_from_dataset and check_answer_tokenization for unit_testing?

**Comment by afansi** [comment]
I dont think we still need those functions.

Il mer 5 giu 2019, 7:42 PM Gaetan Marceau Caron <notifications@github.com>
ha scritto:

**Pull request closed without merging**